R/pkg/R/generics.R

```diff
@@ -439,6 +439,10 @@ setGeneric("covar_samp", function(col1, col2) {standardGeneric("covar_samp") })
 #' @export
 setGeneric("covar_pop", function(col1, col2) {standardGeneric("covar_pop") })
 
+#' @rdname dapply
+#' @export
+setGeneric("dapply", function(x, func, schema = NULL) { standardGeneric("dapply") })
```

Comment:
The two use cases could be done by:
1) combine multiple directly chained functions into single one, this is also useful even we support the no-schema mapPartition()
2) The serializer in JVM should be much faster than that in R. In order to speedup collect for R, maybe another column format (column in R) could be faster than current (rows in R and Java). This could be another topic.

Once we expose the raw format to users, user may see it as public interface, may save the binary as parquet file, it's hard to maintain it in compatible way.